dev/ci/docker/README.md

```diff
@@ -4,10 +4,21 @@ This directory provides Docker images to be used by Coq's CI. The
 images do support Docker autobuild on `hub.docker.com` and Gitlab's
 private registry.
 
-Gitlab CI will build and tag a Docker by default for every job if the
-`SKIP_DOCKER` variable is not set to `false`. In Coq's CI, this
-variable is usually set to `false` indeed to avoid booting a useless
-job.
+Gitlab CI will build and tag a Docker when running a pipeline if either of the
+conditions below is met:
+- If an image with the same name is not available in the
+  [Gitlab container registry](https://gitlab.com/coq/coq/container_registry)
+- If the `SKIP_DOCKER` variable is not set to `false`. In Coq's default CI, this
+  variable is set to `true` precisely to avoid running a lengthy, redundant job.
+
+The first situation can be used to regenerate a fresh Docker image on Gitlab, by
+modifying the `CACHEKEY` value at the first line of the
+[Dockerfile](bionic_coq/Dockerfile) to any unique fresh name, and pushing a
+Github PR with this patch. This will trigger in the corresponding CI a rebuild of the
```

Comment:
You must then manually trigger a Gitlab build with the `SKIP_DOCKER` variable set to “false”.

Reply:
How the hell do you do that?

Reply:
On the pipeline view https://gitlab.com/coq/coq/pipelines there is a big green "run pipeline" button leading to https://gitlab.com/coq/coq/pipelines

Reply:
On that page: https://gitlab.com/coq/coq/pipelines/ you hit the green button (upper right) “Run Pipeline”.

Then you choose the branch that coq-bot has set up for you (pr-NNNNN), named after the number of your PR.

You must define the variables you need.